build.gradle

```diff
@@ -312,6 +312,9 @@ subprojects {
             testCompile libraries.junit,
                     libraries.mockito,
                     libraries.truth
+            testImplementation (libraries.guava_testlib) {
```

Comment:
Should not be done here. Search for usage of `libraries.guava_testlib` and change each of them with exclusion. (census/build.gradle uses `libraries.guava_testlib` but actually it should not)